.github/ISSUE_TEMPLATE/bug_report.yml

```
@@ -0,0 +1,107 @@
+name: Bug report
+description: You've found a bug with Renovate
+labels: ['type:bug', 'status:requirements', 'priority-5-triage']
+body:
+  - type: markdown
+    attributes:
+      value: '## Please do not report any security concerns this way'
+
+  - type: markdown
+    attributes:
+      value: Email renovate-disclosure@whitesourcesoftware.com with any security concerns.
+
+  - type: dropdown
+    id: how-are-you-running-renovate
+    attributes:
+      label: How are you running Renovate?
+      options:
+        - 'WhiteSource Renovate hosted app on github.com'
+        - 'Self hosted'
+    validations:
+      required: true
+
+  - type: dropdown
+    id: platform-select
+    attributes:
+      label: Please select which platform you are using if self-hosting.
+      options:
+        - 'Azure DevOps (dev.azure.com)'
+        - 'Azure DevOps Server'
+        - 'Bitbucket Cloud (bitbucket.org)'
+        - 'Bitbucket Server'
+        - 'Gitea'
+        - 'github.com'
+        - 'GitHub Enterprise Server'
+        - 'gitlab.com'
+        - 'GitLab self-hosted'
+    validations:
+      required: false
+
+  - type: input
+    id: renovate-version-if-selfhosting
+    attributes:
+      label: If you're self-hosting Renovate, tell us what version of Renovate you run.
+      placeholder: '25.56.1'
```

```suggestion
      placeholder: ''
```

Otherwise users will simply use that value without checking